frontend/src/components/UserTables/StudentsTable.tsx

```diff
@@ -66,6 +66,15 @@ const StudentsTable = ({ searchName }: studentTableProps) => {
   const formatDate = (date: string): string =>
     moment(date).format('MM/DD/YYYY');
 
+  const filterStudents = (filter: boolean) =>
```

Comment:
Also, the parameter isn't necessary as we are not going to call this multiple times, and we have everything we need in scope to calculate the result immediately.

ie `const filteredStudents = riders.filter...`

Reply:
Alright. Is there a specific convention that the codebase follows? Arrow functions vs regular functions? Explicit type naming, etc?
eg 
`const filteredStudents = ....` vs `const filteredStudents = () => ....`